appveyor.yml

```diff
@@ -10,14 +10,21 @@ environment:
   matrix:
     - generator: MinGW Makefiles
       CXX_PATH: 'C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin'
+      BUILD_SHARED_LIBS: ON
       ARCH: x64
     - generator: MinGW Makefiles
       CXX_PATH: 'C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin'
+      BUILD_SHARED_LIBS: ON
```

Comment:
Should there be a test for static builds on MinGW?

Reply:
I don't know. The MSVC one seems to be catching the issues with LZMA, so it might be enough. And Appveyor is already very slow ...